web_src/js/components/RepoActionView.vue

```diff
@@ -448,40 +466,71 @@ export function ansiLogToHTML(line) {
 }
 
 .job-group-section .job-brief-list .job-brief-item .job-brief-link {
-  flex-grow: 1;
   display: flex;
+  width: 100%;
 }
 
+@media (max-width: 992px) {
+  .job-group-section .job-brief-list .job-brief-item .job-brief-link {
+    width: 80%;
+  }
+}
 .job-group-section .job-brief-list .job-brief-item .job-brief-link span {
   display: flex;
   align-items: center;
 }
 
+.job-group-section .job-brief-list .job-brief-item .job-brief-link .job-brief-name {
```

Comment:
The selector is so long. Is it necessary to be this long?

Reply:
No.. modified in https://github.com/go-gitea/gitea/pull/24718/commits/05330a1f697e47ecd20bea869a1225124229a7ce